Strict templates in FeaturesUsage failing

The `.demoRawTextCommands` and `.demoDoubleBraces` templates are failing when rendered in `FeaturesUsage`. The templates have `ContentKind.TEXT` but validation fails as the default template content type is `ContentKind.HTML`. The failure breaks the build.

This PR explicitly sets the expected content type to `ContentKind.TEXT`.

```
[INFO] --- exec-maven-plugin:1.3.2:java (java-features-example) @ soy ---
[WARNING] Warning: killAfter is now deprecated. Do you need it ? Please comment on MEXEC-6.
--------------------------------------------------------------------------------
[1. demoComments]
blah blah<br>http://www.google.com<br>
--------------------------------------------------------------------------------
[2. demoLineJoining]
First second.<br><i>First</i>second.<br>Firstsecond.<br><i>First</i> second.<br>Firstsecond.<br>
--------------------------------------------------------------------------------
[3. demoRawTextCommands]
[WARNING] 
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:293)
	at java.lang.Thread.run(Thread.java:724)
Caused by: com.google.template.soy.tofu.SoyTofuException: Expected template to be kind="html" but was kind="text": soy.examples.features.demoRawTextCommands
	at com.google.template.soy.tofu.internal.BaseTofu$RendererImpl.enforceContentKind(BaseTofu.java:527)
	at com.google.template.soy.tofu.internal.BaseTofu$RendererImpl.render(BaseTofu.java:499)
	at com.google.template.soy.tofu.internal.BaseTofu$RendererImpl.render(BaseTofu.java:486)
	at com.google.template.soy.examples.FeaturesUsage.execMain(FeaturesUsage.java:147)
	at com.google.template.soy.examples.FeaturesUsage.main(FeaturesUsage.java:91)
	... 6 more

```

--- a/java/src/com/google/template/soy/examples/FeaturesUsage.java
+++ b/java/src/com/google/template/soy/examples/FeaturesUsage.java
@@ -23,6 +23,7 @@ import com.google.inject.Guice;
 import com.google.inject.Injector;
 import com.google.template.soy.SoyFileSet;
 import com.google.template.soy.SoyModule;
+import com.google.template.soy.data.SanitizedContent;
 import com.google.template.soy.data.SoyListData;
 import com.google.template.soy.data.SoyMapData;
 import static com.google.template.soy.examples.FeaturesSoyInfo.DEMO_AUTOESCAPE_TRUE;
@@ -144,7 +145,10 @@ public class FeaturesUsage {
     System.out.println(tofu.newRenderer(DEMO_LINE_JOINING).setMsgBundle(msgBundle).render());
 
     writeExampleHeader("demoRawTextCommands");
-    System.out.println(tofu.newRenderer(DEMO_RAW_TEXT_COMMANDS).setMsgBundle(msgBundle).render());
+    System.out.println(tofu.newRenderer(DEMO_RAW_TEXT_COMMANDS)
+      .setMsgBundle(msgBundle)
+      .setContentKind(SanitizedContent.ContentKind.TEXT)
+      .render());
 
     writeExampleHeader("demoPrint");
     System.out.println(tofu.newRenderer(DEMO_PRINT)
@@ -244,6 +248,7 @@ public class FeaturesUsage {
                                  DEMO_DOUBLE_BRACES.SET_MEMBERS,
                                      ImmutableList.of(2, 3, 5, 7, 11, 13)))
          .setMsgBundle(msgBundle)
+         .setContentKind(SanitizedContent.ContentKind.TEXT)
          .render());
 
     // The Hebrew in the following example comes out as question marks in the output because

